subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -453,6 +453,47 @@ config BT_CTLR_LLCP_CONN
 	  more than this number of connections simultaneously may cause
 	  instabilities.
 
+if !BT_LL_SW_SPLIT_LLCP_LEGACY
+config BT_CTLR_LLCP_PER_CONN_TX_CTRL_BUF_NUM
+	int "Number of tx control buffers to be reserved per connection"
+	default 1
+	range 0 4
+	help
+	  Set the number control buffers that is to be pre allocated per connection
+	  This defines the minimum number of buffers available for any connection
+	  Setting this to non zero will ensure a connection will always have access
+	  to buffer(s) for control procedure TX
+
+config BT_CTLR_LLCP_COMMON_TX_CTRL_BUF_NUM
+	int "Number of tx control buffers to be available across all connections"
+	default BT_MAX_CONN
+	range 1 255
+	help
+	  Set the number control buffers that is to be available for tx.
+	  This defines the size of the pool of tx buffers available
+	  for control procedure tx. This pool is shared across all
+	  procedures/connections with allocation through a fifo queue
```

Comment:
If every connection has it's theoretical maximum number of buffers, then there is no need for a shared common pool of buffers, correct?

Reply:
Correct, I'll change the range to include 0.
Also I was in doubt as to what to make the default? BT_MAX_CONN is probably not so good, as it implies a relation, and there is none!